cmd/snap/cmd_services.go

```diff
@@ -124,6 +124,11 @@ func (s *svcStatus) Execute(args []string) error {
 		return err
 	}
 
+	if len(services) == 0 {
+		fmt.Fprintln(Stderr, i18n.G("No snaps providing services are installed."))
```

Comment:
that's a little clunky, indeed. My suggestion isn't much better, so I'll defer to @degville unless he comes up empty.

Reply:
Hm, a thought, since the command is `snap services`, maybe something like this could work `There are no services provided by snaps` (or `... installed snaps`). wdyt?

Reply:
@bboozzoo I think you're right - yes, that's better (I'd go with *installed snaps*).